### ISSUE TITLE
Clarify logging wording wrt auto-rebased test shards

### ISSUE THREAD

**Issue by cathyxz** [ampproject/amphtml#21815]
Today I learned that our test shards are individually auto-rebasing provided that there are no merge conflicts. This is insanely awesome, which is why we should make that clear to readers of our said logs so they can just rerun the shard instead of manually rebasing and causing more than necessary tests to run (like what I did to all the renovate PRs today). 

The log currently reads like this: 
```
build.js: Running build.js...
build.js: origin/master is currently at commit 9eccf04
build.js: Testing the following changes at commit 98a88e7
... // REALLY LONG GIT DIFF

build.js: Commit log since branch 2019-03-22-ClosureCompiler was forked from master at 7b6c5a7:
... // REALLY LONG COMMIT LOG
* 7b6c5a7 Aaron Labiaga - Additional message in validation pass to follow up with CORS testing (#21313) (20 hours ago)
```

It's just a bit hard to connect the dots here since people don't naturally memorize commit hashes and the diffs / commit logs can be long. Just somewhere in the beginning, you can add a line to say these changes are applied in commit X on commit Y off of latest master. Or you can log below the failure message that "if this test has since been fixed, rerunning this shard will automatically rebase".  

Basically if we have a fantastic new build feature, devs should know. ;)

**Comment by rsimha** [comment]
I think this messaging will be more visible in the "AMP test status" details page. For example, [here](https://github.com/ampproject/amphtml/pull/21811/checks?check_run_id=98451785).

Assigning to @danielrozenberg who can add the line there.

**Comment by danielrozenberg** [comment]
I agree with @rsimha, I'll keep this in mind as I work on the test status bot

**Comment by rsimha** [comment]
This is being tracked at https://github.com/ampproject/amp-github-apps/issues/144.

Closing this out.
